src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -290,3 +298,70 @@ def _convert_meta(self, source: Node, concept: Concept):
                     value=synonym.val,
                 )
             )
+
+
+@dataclass
+class OboGraphToFhirNpmConverter(OboGraphToFhirJsonConverter):
```

Comment:
@ShahimEssaid Attached is the output from the first unit test taking a pass at this. Let me know what you think! [CodeSystem-sequence-ontology.tgz](https://github.com/INCATools/ontology-access-kit/files/11165795/CodeSystem-sequence-ontology.tgz)

Reply:
@ShahimEssaid Not sure if you had a chance to review this, but marking this as resolved to declutter this PR.